controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -473,3 +473,47 @@ def test_create_snapshot_success(self):
 
         self.assertEqual(snapshot.snapshot_name, volume.name)
         self.assertEqual(snapshot.id, self.array._generate_volume_scsi_identifier(volume.id))
+
+    def test_create_snapshot_not_valid(self):
```

Comment:
shouldn't this be in https://github.com/IBM/ibm-block-csi-driver/pull/163/?

Reply:
forgot then..

Reply:
resolving due to branch merge+delete (and not actual fix)